homeassistant/components/prosegur/alarm_control_panel.py

```diff
@@ -59,6 +60,17 @@ def __init__(self, contract: str, auth: Auth) -> None:
         self._attr_name = f"contract {self.contract}"
         self._attr_unique_id = self.contract
 
+    @property
+    def device_info(self) -> DeviceInfo:
```

Comment:
This property could be replaced by the attribute I think (self._attr_device_info)

Reply:
Done